wasm/jsapi/instance/caching.any.js

```diff
@@ -0,0 +1,67 @@
+// META: global=jsshell
+// META: script=/wasm/jsapi/wasm-module-builder.js
+
+promise_test(async t => {
+  const builder = new WasmModuleBuilder();
+  builder.addImportedMemory("module", "memory", 0, 128);
+  builder.exportMemoryAs("exported");
+  const buffer = builder.toBuffer()
+
+  const memory = new WebAssembly.Memory({ initial: 64, maximum: 128 });
```

Comment:
```suggestion
  const memory = new WebAssembly.Memory({ initial: 0, maximum: 128 });
```
Matches the memory import declaration above